content/en/docs/tasks/tls/manual-rotation-of-ca-certificates.md

```diff
@@ -0,0 +1,136 @@
+---
+title: Manual rotation of CA certificates
+content_template: templates/task
+reviewers:
+- 
+---
+
+{{% capture overview %}}
+
+This page shows how to manually rotate certificate authority (CA) certificates.
+
+{{% /capture %}}
+
+
+{{% capture prerequisites %}}
+
+{{< include "task-tutorial-prereqs.md" >}} {{< version-check >}}
+
+For more information about authentication in Kubernetes, see "[Authenticating](/docs/reference/access-authn-authz/authentication)".
+For more information about best practices for CA certificates, see "[Single root CA](docs/setup/best-practices/certificates/#single-root-ca)".
+
+{{% /capture %}}
+
+{{% capture steps %}}
+
+There are two ways to rotate or replace CA certificates.
+
+## Replace the cluster (recommended)
+
+This is the safest approach to rotate or replace the CA certificates.
+
+1. Create a new cluster with new CA certificates.
+
+2. Migrate the workloads to the new cluster.
+
+3. Remove or destroy the old cluster.
+
+## Rotate the CA certificates manually
+
+{{< caution >}}
+Kubeadm does not directly support rotating or replacement of CA certificates. Make sure to back up your certificate directory along with confs and any other necessary files.
```

Comment:
```suggestion
The `kubeadm` tool does not directly support rotating or replacement of CA certificates. Make sure to back up your certificate directory along with configuration files and any other necessary data.
```

Reply:
This line is removed and added on kubeadm-certs page as @neolit123 suggested.